content/swiftui/concepts/views/terms/Label/label.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Label'
+Description: 'A label is a standard user interface component that includes an icon and a title.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Label'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **`label`** is a standard user interface component that includes an icon and a title. It appears in many different context like collections, lists or buttons, to name a few.
+
+## Syntax
+
+```pseudo
+Label("Lightning", systemImage: "bolt.fill")
+}
+```
+
+The `label` can be a stand-alone component. It has modifiers where it can display just the icon, text, both or automatically adjust based on the space available.
```

Comment:
```suggestion
A `Label` can be a stand-alone component. It has modifiers where it can display just the icon, text, both or automatically adjust based on the space available.
```